doc/modules/model_evaluation.rst

```diff
@@ -6,6 +6,113 @@
 Metrics and scoring: quantifying the quality of predictions
 ===========================================================
 
+.. _which_scoring_function:
+
+Which scoring function should I use?
+====================================
+
+Before diving into the details of the many scores and metrics, we want
+to give some guidance, inspired by statistical decision theory, on the choice
+of **scoring functions** for **supervised learning**, see [Gneiting2009]_:
+
+- *Which scoring function should I use?*
+- *Which scoring function is a good one for my task?*
+
+In a nutshell, if the scoring function is given, e.g. in a kaggle competition
+or in a business context, use that one.
+If you are free to choose, it starts by considering the ultimate goal and application
+of the prediction. It is useful to distinguish two steps:
+
+* Predicting
+* Decision making
+
+**Prediction:**
+Usually, the response variable :math:`Y` is a random variable, in the sense that there
+is *no deterministic* function :math:`Y = g(X)` of the features :math:`X`.
+Instead, there is a probability distribution :math:`F` of :math:`Y`.
+One can aim to predict the whole distribution, known as *probabilistic prediction*,
+or---more the focus of scikit-learn---issue a *point prediction* (or point forecast)
+by choosing a property or functional of that distribution :math:`F`.
+Typical examples are the mean (expected value), the median or a quantile of the
+response variable :math:`Y` (conditionally on :math:`X`).
+
+Once that is settled, use a **strictly consistent** scoring function for that
```

Comment:
Can we add in parenthesis, what "strictly consistent" means?

Reply:
The meaning of it is just a few lines below with the quote about "truth telling".

Note that I once gave the mathematical details, but removed them in https://github.com/scikit-learn/scikit-learn/pull/11430/commits/5517d0d204bc633a6542089b54c75feecd5c03e3. It's just too long, better you look up in the linked papers.

Reply:
https://github.com/scikit-learn/scikit-learn/pull/11430/commits/428a3b5b2492d8dfedb493aee7296dccce53f140